app/controllers/partner_requests_controller.rb

```diff
@@ -7,10 +7,14 @@ def index
   end
 
   def new
-    @partner_request = PartnerRequest.new
-    @valid_items = DiaperBankClient.get_available_items(current_partner.diaper_bank_id)
-    @valid_items = @valid_items.map { |item| [item["name"], item["id"]] }
-    @partner_request.item_requests.build # required to render the empty items form
+    if current_partner.partner_status == "Approved"
```

Comment:
Not blocking, but I wonder if this should live in a more explicit authorization down the road. I like how it is for now though!

Reply:
agreed. a refactoring of `partner_status` into an enum would be a good idea too and might make a cancan/pundit like authorization easier.